native-sql-engine/core/src/main/scala/com/intel/oap/extension/ColumnarOverrides.scala

```diff
@@ -204,17 +204,17 @@ case class ColumnarPreOverrides() extends Rule[SparkPlan] {
       logDebug(s"Columnar Processing for ${plan.getClass} is currently supported.")
       plan
 
-    case plan: CustomShuffleReaderExec if columnarConf.enableColumnarShuffle =>
+    case plan: AQEShuffleReadExec if columnarConf.enableColumnarShuffle =>
```

Comment:
Add guard logic to check the type on shim layer.
plan.child, plan.partitionSpecs should be obtained on shim layer.